### ISSUE TITLE
plot_model of lmer model no longer recognizing grouping variables

### ISSUE THREAD

**Issue by abaimel** [strengejacke/sjPlot#390]
Hi there, 

I used to be able to do the following to plot the predictions of X on Y by GROUP from a lmer model. 

model <- lmer(X ~ 1 + X + (1 + X|GROUP))
plot_model(model, type="pred", pred.type="re", terms=c("X", "GROUP")) 

This call is now producing the following error and not recognizing the grouping variable
"Error in `[.data.frame`(as.data.frame(mm), terms) : 
  undefined columns selected".

I've replicated this error with different datasets, and different grouping variables. Any intel? Happy to provide more details should you need it. I'm currently running all the latest builds installed from github.

Cheers!

**Comment by strengejacke** [comment]
It looks like you're trying to predict values from a term that is not a fixed effect (`GROUP`), which is not possible. You can _condition_ on random effects with `pred.type = "re"`, but you can calculate marginal effects of `X` by the levels of `GROUP`.

**Comment by abaimel** [comment]
Understood ~ yes, that makes complete sense. Thanks.

However, that leaves me wondering what was being produced when this code functioned - I imagine I couldn't have been getting more than just a plot of the random intercepts and slopes by grouping variable. Is there such a call in the current plot_model() function so that I can compare against older plots where this worked (I might be crazy but can't find it!).

**Comment by strengejacke** [comment]
Sorry, typo, should read "can not"...

**Comment by strengejacke** [comment]
You can plot random effects, with `type = "re"`.

**Comment by strengejacke** [comment]
> Is there a "rs.ri" type for plot_model?
No, not yet. The "rs.ri"-type was misleading and lead to wrong inferences, because actually it was not suitable to calculate the effects of random intercept/slopes in the way the function did, that's why I removed that option and why there's currently no replacement.
